lib/libc/atexit.c

```diff
@@ -0,0 +1,24 @@
+#define MAX_ATEXIT 128
+
+static void (*AtExit[MAX_ATEXIT])(void);
+static unsigned NumAtExit;
+
+void klee_report_error(const char *, int, const char *, const char *);
+static void RunAtExit(void) __attribute__((destructor));
+static void RunAtExit(void) {
+    unsigned i;
+
+    for (i = NumAtExit - 1; i < MAX_ATEXIT; --i) AtExit[i]();
```

Comment:
Please get rid of this underflow. It works correctly, but it may bring unnecessary work to verifiers -- more than when using just the explicit comparison to 0.

Reply:
Also, when you're at that, put the body of the cycle on the new line, to be consistent with the rest of the files.